uploadscript/script.js

```diff
@@ -47,7 +50,8 @@ const performUpload = async () => {
                 await uploadFileToFirebaseStorage(incorrectFolder + "incorrect_assessment1.jpg", wordRef.id + "/incorrect_assessment1.jpg"),
                 await uploadFileToFirebaseStorage(incorrectFolder + "incorrect_assessment2.jpg", wordRef.id + "/incorrect_assessment2.jpg"),
                 await uploadFileToFirebaseStorage(incorrectFolder + "incorrect_assessment3.jpg", wordRef.id + "/incorrect_assessment3.jpg"),
-            ]
+            ],
+            assessmentPrompt: await uploadFileToFirebaseStorage(promptFolder + "assessmentPrompt.mp3", wordRef.id + "/assessmentPrompt.mp3")
```

Comment:
Can we call this assessment.mp3? assessmentPrompt seems unnecessary since it is in the `prompts` folder.